indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -247,7 +247,7 @@
         :let [key (key humanized-field)
               key-name (-> key
                            name
-                           (s/replace #"(\.humanized(_?2)?|-sn|-id)" ""))
+                           (s/replace #"((_\.)?humanized(_?2)?|-sn|-id)" ""))
```

Comment:
Are the changes to this file something we'd also want to add to master for 1.6 as well or do we only want these for the 7.x upgrades?